sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -171,3 +171,40 @@ def test_Beam():
     raises(ValueError, lambda: b4.apply_load(-3, 0, -1, end=3))
     with raises(TypeError):
         b4.variable = 1
+
+
+def test_beam_units():
+    from sympy.physics.units import meter, kilogram, newton, second, kilo
+    E, I = symbols('E, I')
+    b = Beam(3*meter, E*newton/meter**2, I*meter**4)
+    b.apply_load(8*kilo*newton, 1*meter, -1)
+    b.apply_load(R1*kilo*newton, 0*meter, -1)
+    b.apply_load(R2*kilo*newton, 3*meter, -1)
+    b.apply_load(12*kilo*newton*meter, 2*meter, -2)
+    b.bc_deflection = [(0*meter, 0*meter), (3*meter, 0*meter)]
+    b.solve_for_reaction_loads(R1, R2)
+
+    p = b.reaction_loads
+    q = {R1: -28/3, R2: 4/3}
```

Comment:
these should have units, no?

Reply:
I don't think it should have units as the applied force is `b.apply_load(R1*kilo*newton, 0*meter, -1)`
so R1 should be only a number and not a quantity.

Reply:
But R1 and R2 are loads and should be in units of force.